.github/workflows/workflow-run.yml

```diff
@@ -1,31 +1,66 @@
+# ------------- [General Knowledge] -------------
+# Workflow syntax table
+# Doc: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions
+
+# ------------- [General Knowledge] -------------
+# Github Actions has two collections of variables:
+# (1) Contexts:
+#     There're different kinds of context object, e.g., github, env, jobb, etc.
+#     Doc: https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#contexts
+# (2) Default environment variables:
+#     Doc: https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables
+#
+# You can use them in the `if` condition to prevent a job from running unless a condition is met.
+# Or use them to set up a env variable for a job/ step.
+# Or use them in a job running step directly.
+# Doc: https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#about-contexts-and-expressions
+
 name: workflow run
 
+# Since dependabot doesn't have write permission in event "pull_request", we have two options:
+# (1) Event "pull_request_target"
+#     This event runs in the context of the base of the pull request, rather than in the merge commit as the pull_request event does.
+# Doc: https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request_target
+# (2) Event "workflow_run"
+#     This event occurs when a workflow run is requested or completed, and allows you to execute a workflow based on the finished result of another workflow.
+# Doc: https://docs.github.com/en/actions/reference/events-that-trigger-workflows#workflow_run
+# See [detail](https://github.blog/changelog/2021-02-19-github-actions-workflows-triggered-by-dependabot-prs-will-run-with-read-only-permissions/)
+
+# To test whether this workflow works or not,
+# just re-open a PR from dependabot that might triggers dependabot to force push a new commit (say commit id `_abcde_`),
```

Comment:
Or you can ask dependabot to create a pr from the [dependabot alert list](https://github.com/marilyn79218/react-lazy-show/security/dependabot) in your repo, [doc](https://docs.github.com/en/code-security/supply-chain-security/managing-vulnerabilities-in-your-projects-dependencies/viewing-and-updating-vulnerable-dependencies-in-your-repository#viewing-and-updating-vulnerable-dependencies)